src/app/stores/AppConfigStore.js

```diff
@@ -0,0 +1,10 @@
+import alt from '../alt';
+import appConfig from '../../../appConfig';
+
+class AppConfigStore {
```

Comment:
So this is a way of storing the appConfig in a dedicated `Store`? Just asking, but why not just import `appConfig` where it's needed? That question is just for my understanding.

Reply:
Yeah, I expect we may be asked to change this. It's mostly to avoid putting the appConfig on the window, and also to make use of the existing setup we have for loading data.